torch/csrc/jit/codegen/cuda/graph_fuser.cpp

```diff
@@ -839,7 +839,7 @@ struct CudaGraphFuser {
           fmap(tensor_inputs, [&](Value* v) { return shape_of.at(v); });
       AT_ASSERT(!shapes.empty());
```

Comment:
Note to myself:
We should assert/throw a warning here on (n->outputs().size() == 1). Meanwhile, add a special rule for dropout.